src/elements/content-sidebar/messages.js

```diff
@@ -117,6 +117,11 @@ const messages = defineMessages({
         defaultMessage: 'Modify General Task',
         description: 'modal title for when editing an existing general task',
     },
+    expandBoxAI: {
+        id: 'be.contentSidebarExpandBoxAI',
```

Comment:
Can we namespace this to be similar to the other messages? Maybe `be.contentSidebar.boxAI.expand`?